src/main/java/com/group12/LaunchInterceptorConditionCollection.java

```diff
@@ -29,11 +29,74 @@ public boolean LIC0(List<Point> points, double length) throws IllegalArgumentExc
         return false;
     }
 
+    /**
+     * LIC #1 checks if there exists at least one set of three consecutive data points that cannot all be contained
+     * within or on a circle of the given <b>radius</b> (RADIUS1).
+     *
+     * @param points list of radar echos ({@link Point})
+     * @param radius has to be greater than 0 (zero)
+     * @return true if at least one set of three consecutive data points forms a circle with radius greater than
+     * given <b>radius</b>, false otherwise.
+     * @throws IllegalArgumentException is thrown if <b>radius</b> is less than 0 (zero)
+     */
+    public boolean LIC1(List<Point> points, double radius) throws IllegalArgumentException {
+        if (radius < 0) {
+            throw new IllegalArgumentException("Radius cannot be less than zero");
+        }
```

Comment:
What do you think of adding a check for `points` list as well? What happens if the input is `null` or if the list contains only zero, one or two elements? In this case, should we throw an `IllegalArgumentException` or should we return true/false? I guess it's up to us to decide, what do you think?

Reply:
In my PR, I decided to throw `IllegalArgumentException`. But if we decide upon something else, I can change mine too.